docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -0,0 +1,284 @@
+---
+slug: getting-started-with-semgrep-app
+append_help_link: true
+title: Getting started with Semgrep App
+description: "Get started with Semgrep App to scan for security vulnerabilities on cloud repositories hosted on GitHub and GitLab."
+---
+
+# Scanning cloud repositories with Semgrep App
+
+Semgrep App enables you to run scans on multiple repositories by integrating with your GitHub or GitLab-hosted account. Scans are powered by **rules**. A Semgrep rule encapsulates pattern-matching logic and data-flow analysis used to find code violations, security issues, outdated libraries, and other issues.
+
+
+![Diagram of Semgrep App flow](../img/semgrep-app-diagram.png "Diagram of Semgrep App flow")
+
+
+Many rules are available from [Semgrep Registry](https://semgrep.dev/r), an open-source, community-driven repository of rules. Write your own rules to create general security checks through the Registry while customizing Semgrep to work for your team's specific secret-keeping and API practices.
+
+After performing a scan, use Semgrep App's Rule Board feature to prevent pull requests (PRs) or merge requests (MRs) from merging until findings are resolved. This helps prevent vulnerable code from shipping to widely-accessible environments, such as production or staging servers.
+
+Semgrep App enables you to scan whole repositories on a cloud. You can try Semgrep CLI for local scans as well. For more information, see [Getting started with Semgrep CLI](https://semgrep.dev/docs/getting-started/).

```

Comment:
What does it mean to "scan whole repositories on a cloud"? I could take this to mean code is uploaded to Semgrep App, although the code never leaves a user's environment. Semgrep App enables you to deploy, configure, and manage Semgrep running in your CI environment. Would you be open to language more like this?

Reply:
Have added this in!